pbf2json.go

```diff
@@ -108,15 +127,23 @@ func run(d *osmpbf.Decoder, db *leveldb.DB, config Settings){
           }
 
           // ----------------
-          // handle tags
+          // handle conditions
           // ----------------
 
-          if !hasTags(v.Tags) { break }
-
-          v.Tags = trimTags(v.Tags)
-          if containsValidTags( v.Tags, config.Tags ) {
-            onNode(v)
+          if len(config.Ids) != 0 {
+            i := sort.SearchStrings(config.Ids, strconv.FormatInt(v.ID,10))
```

Comment:
I'm not sure if this is doing what you expected. The [docs](https://golang.org/pkg/sort/#SearchStrings) are a little cryptic  when they say:

> SearchStrings searches for x in a sorted slice of strings and returns the index as specified by Search. The return value is the index to insert x if x is not present (it could be len(a)). The slice must be sorted in ascending order.

I did a little experiment and it seems that `SearchStrings()` returns the position in the sorted set where the new record should be inserted, it's not useful in confirming that the id actually exists.

```go
var ids = []string{"100", "200", "300"}

fmt.Println(sort.SearchStrings(ids, "105"))
fmt.Println(sort.SearchStrings(ids, "400"))
```

```bash
$ go run searchstrings.go 
1
3
```